mono/mini/mini-arm64.c

```diff
@@ -2040,7 +2045,7 @@ mono_arch_flush_icache (guint8 *code, gint size)
 #ifndef MONO_CROSS_COMPILE
 #if __APPLE__
 	sys_icache_invalidate (code, size);
-#else
+#elif !HOST_WIN32
```

Comment:
You are missing Win32 path here. Use [FlushInstructionCache](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-flushinstructioncache).